Makefile

```diff
@@ -16,3 +16,6 @@ gen:
 		--with-libra-types "testsuite/generate-format/tests/staged/libra.yaml" \
 		--target-source-dir ".." \
 		"language/stdlib/compiled/transaction_scripts/abi"
+
+protoc:
+	protoc  --go_out=. --go_opt=paths=source_relative ./librajsonrpctypes/libra-jsonrpc-types.proto
```

Comment:
do we need to fix the headers in the generated code after running this?

Reply:
no, current package and option go_package is used for generate correct result.